e2e/nx-misc/src/workspace.test.ts

```diff
@@ -27,12 +27,12 @@ describe('Workspace Tests', () => {
 
   afterAll(() => cleanupProject());
 
-  describe('@nrwl/workspace:library', () => {
+  describe('@nrwl/js:library', () => {
```

Comment:
These should probably be moved to `e2e-js`

Reply:
I moved and combined them with existing tests in the js project